.gitmodules

```diff
@@ -1,10 +1,10 @@
 [submodule "gFTL"]
 	path = extern/gFTL
-	url = https://github.com/Goddard-Fortran-Ecosystem/gFTL.git
```

Comment:
FYI these submodule links point to a different repo and it looks like this was meant to be changed before merged.

Reply:
I'll submit a pull request to revert that, if you'd like.

Reply:
There are some PRs in those submodules that need to be merged first. But due to some circular dependencies, they were a little painful. They might actually be easier now this is merged

Reply:
Ah ok thanks for the additional context.